### PR TITLE
Create pd-control.md

### PR DESCRIPTION
**Pull request by Wenting0905**
PTAL @shenli

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @disksing

File: op-guide/pd-control.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# PD Control instructions
+
+As a command line tool of PD, PD Control is used for obtaining the cluster state information and tuning the cluster.
+
+## Source code compiling
+
+1. [*Go*](https://golang.org/) Version 1.7 or later
+2. In the PD root directory, use `make` command to compile and generate bin/pd-ctl
+
+## A sample
```
Comment (shenli) [inline op-guide/pd-control.md]: A sample -> Usage

File: op-guide/pd-control.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+```bash
+export PD_ADDR=http://127.0.0.1:2379
+./pd-ctl
+```
+
+## command line flags (flags)
+
+### \-\-pd,-u
+
++ Specify PD address
```
Comment (shenli) [inline op-guide/pd-control.md]: Specify PD address -> PD address

File: op-guide/pd-control.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    ./pd-ctl -u 127.0.0.1:2379
+
+use environment variables:
+
+```bash
+export PD_ADDR=http://127.0.0.1:2379
+./pd-ctl
+```
+
+## command line flags (flags)
```
Comment (shenli) [inline op-guide/pd-control.md]: command line flags (flags) -> command line flags

File: op-guide/pd-control.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+## command (command)
+
+### store [delete] \<store_id\>
+
+This is used for displaying the store information or deleting the specified store.
+
+Sample:
+
+```bash
+>> store            // Display all store information
```
Comment (shenli) [inline op-guide/pd-control.md]: all store -> stores?

File: op-guide/pd-control.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+{
+  "count": 3,
+  "stores": [...]
+}
+>> store 1          // Get the store with a store id 1
+  ......
+>> store delete 1   // Put the store with a store id 1 offline
```
Comment (shenli) [inline op-guide/pd-control.md]: Put -> Make

File: op-guide/pd-control.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+Sample:
+
+```bash
+>> region                               //　Display all region information
+{
+  "count": 1,
+  "regions": [......]
+}
+
+>> region 2                             // Display the information with a region id 2
```
Comment (shenli) [inline op-guide/pd-control.md]: Display the information of region 2.

File: op-guide/pd-control.md (hunk 1 above)
Comment (disksing) [inline op-guide/pd-control.md]: Will 'example' be better than 'sample'?

File: op-guide/pd-control.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+{
+  "members": [......]
+}
+>> member leader                        // Display the information of leader
+{
+  "name": "pd",
+  "addr": "http://192.168.199.229:2379",
+  "id": 9724873857558226554
+}
+>> member delete pd2                    // Put "pd2" offline
```
Comment (shenli) [inline op-guide/pd-control.md]: Put -> Make

**Comment by Wenting0905** [comment]
PTAL @shenli @siddontang @disksing

File: op-guide/pd-control.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
++ Use single command line mode (not entering readline)
++ Default value: false
+
+## command (command)
+
+### store [delete] \<store_id\>
+
+This is used for displaying the store information or deleting the specified store.
+
+Sample:
```
Comment (disksing) [inline op-guide/pd-control.md]: Replace all 'Sample' to 'Usage' to make them consistent.

**Comment by Wenting0905** [comment]
PTAL @shenli @siddontang @disksing

**Comment by disksing** [comment]
LGTM

File: op-guide/pd-control.md [hunk 9]
```diff
@@ -0,0 +1,206 @@
+# PD Control instructions
+
+As a command line tool of PD, PD Control is used for obtaining the cluster state information and tuning the cluster.
+
+## Source code compiling
```
Comment (siddontang) [inline op-guide/pd-control.md]: we don't want to user build pd-ctl, `pd-ctl` exists in the release binary already.

File: op-guide/pd-control.md [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+### \-\-detach,-d
+
++ Use single command line mode (not entering readline)
++ Default value: false
+
+## command (command)
+
+### store [delete] \<store_id\>
+
+This is used for displaying the store information or deleting the specified store.
```
Comment (siddontang) [inline op-guide/pd-control.md]: I think we can remove all `This is used for`, using `Display the store...` is enough.

**Comment by Wenting0905** [comment]
PTAL @siddontang

**Pull request merged**